manifests/j/JetBrains/CLion/EAP/223.4884.72/JetBrains.CLion.EAP.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 $debug=QUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: JetBrains.CLion.EAP
+PackageVersion: 223.4884.72
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
```

```suggestion
UpgradeBehavior: uninstallPrevious
```
Shouldn't this be set to `uninstallPrevious` like the previous versions?